src/admin-views/widgets/components/radio.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Admin View: Widget Radio Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/radio.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string       $label   Label for the radio group.
+ * @var string       $value   Value for the radio.
+ * @var string       $name    Name attribute for the radio.
+ * @var array<array> $options Data for the individual radio buttons.
+ */
+if ( ! empty( $label ) ) : ?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 7d12f7ba348d04a4cc88df039a2289d8</sub>